dynamicEDT3D/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
-PROJECT(dynamicEDT3D)
+cmake_minimum_required(VERSION 3.5)
+PROJECT( dynamicEDT3D )
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
 include(CTest)
 
+find_package(ament_cmake REQUIRED)
```

Comment:
Is there no way with ament to have it build externally without modifying the original `CMakeLists`? This would break "normal" building with CMake without ament. For catkin, it is built as an external project without changes to the CMakeLists and works fine.